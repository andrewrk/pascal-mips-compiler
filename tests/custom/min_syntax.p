program A;
class A begin
end
.
