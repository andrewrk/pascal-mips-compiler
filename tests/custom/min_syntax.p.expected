ERROR: line 2: main class "A" must have a parameterless constructor
